test_stubdefaulter.py

```diff
@@ -48,10 +49,15 @@ class FooEnum(str, enum.Enum):
 
 def strenum_default(x=FooEnum.FOO):
     return str(x)
+
+if sys.version_info >= (3, 8):
+    def pos_only(x=5, /):
```

Comment:
```suggestion
    def pos_only(x=5):
```
Seems like the easiest solution. Nothing is verifying that the stub used in the test is precisely correct.

Reply:
I guess...!

Reply:
You'll also have to remove the version check or stubdefaulter won't add the default when run under 3.7. (Obviously there's other ways to fix if you prefer.)

Reply:
oh, of course